config/database.yml

```diff
@@ -17,13 +17,13 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  # For details on connection pooling, see Rails configuration guide
-  # https://guides.rubyonrails.org/configuring.html#database-pooling
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: admin
```

Comment:
Check, because the database.yml file is being updated, and that can cause problems in the configuration of the other users.

![imagen](https://user-images.githubusercontent.com/12243331/102280863-838d9080-3ef3-11eb-9c4d-a4d63a0d72fb.png)